apps/api/src/main.ts

```diff
@@ -8,11 +8,16 @@ import { HttpExceptionFilter } from './common/http-exception.filter';
 import { JsendFormatter } from './common/jsend-formatter';
 
 const logDir = 'logs';
+const uploadDir = 'uploads';
```

Comment:
This  is incorrect. What if the env value is added?
You are supposed to get the upload folder path and create the folder there.
Here you are only doing it for default path.

Reply:
Updated the condition to use env value for folder path. Also tested for conditions when env value is missing.